mlir/lib/Dialect/Vector/Transforms/VectorLinearize.cpp

```diff
@@ -27,6 +27,10 @@
 using namespace mlir;
 
 static bool isLessThanTargetBitWidth(Operation *op, unsigned targetBitWidth) {
+  // For BW-0, all operations are legal
+  if (targetBitWidth == 0)
```

Comment:
`targetBitWidth = std::numeric_limits<unsigned>::max()` is used in places, please consolidate.

Reply:
I briefly looked at it. I'm ok with that change, but can we commit this first if possible?

Reply:
I have reached a point where I need to land this to make progress with other development, I hope that's ok @nbpatel 

It should be easy to absorb the changes (just remove code related to bitwidth)

Reply:
yes go ahead and merge it